academic-services/knowledge-exploration-service/how-to-delete-api-instances.md

```diff
@@ -0,0 +1,36 @@
+---
+title: How to delete a MAKES API deployment
+description: Step-by-step guide for deleting a Microsoft Academic Knowledge Exploration Service(MAKES) deployment using azure portal.
+ms.topic: tutorial
+ms.date: 02/06/2020
+---
+
+# How to delete a MAKES deployment
+
+ Step-by-step guide for deleting a MAKES deployment created from MAKES command line tool. For more information on deploying a MAKES API instance, visit [Create an API Instance](get-started-create-api-instances.md)
+
+## Prerequisites
+
+- A MAKES API instance deployment
+
+## Find the resource group associated with the MAKES API instance
```

Comment:
We should have a list of all the resources that are created by the tool in this document.

Reply:
| Item | Description | Count |
|-------|-------|-------|
| VM Scale Sets | blah, blah | 1 |
| MAKES VM's | blah, blah | depends on how many were deployed |
| Public IP address | blah, blah | 1 for each VM deployed |

.....

---
In reply to: [376107674](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/105#discussion_r376107674) [](ancestors = 376107674)

Reply:
I disagree that that information should reside in this document. We *should* have this information, but it should probably be in a more general reference page for the MAKES service (which doesn't currently exist)

---
In reply to: [376605915](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/105#discussion_r376605915) [](ancestors = 376605915,376107674)